src/test/kotlin/org/rust/ide/inspections/RsInspectionsTestBase.kt

```diff
@@ -5,18 +5,29 @@
 
 package org.rust.ide.inspections
 
+import com.intellij.codeInspection.InspectionProfileEntry
+import com.intellij.testFramework.InspectionTestUtil
 import org.rust.TestProject
 import org.rust.ide.annotator.RsAnnotationTestBase
 
 abstract class RsInspectionsTestBase(
-    val inspection: RsLocalInspectionTool
+    private val inspectionClass: Class<out InspectionProfileEntry>
```

Comment:
Consider using `KClass`.
And maybe extract it to another PR? 72 changed files looks scary 😨

Reply:
Okay. I'll extract these changes into separate PR and merge it before this one if you think it's better